iree/turbine/dynamo/backends/base.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2025 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from ..passes import turbine_cpu_pass_pipeline
+from ...transforms.general.custom_op_expansion import ExpandCustomOpsPass
+from iree.turbine.runtime.launch import Launchable
+from iree.turbine.support.logging import aot_logger as logger
+
+import torch
+from torch._dynamo.backends.common import aot_autograd
+from functorch.compile import make_boxed_func
+from iree.turbine.support.ir_imports import (
+    PassManager,
+)
+from iree.compiler.extras.fx_importer import (
+    FxImporter,
+)
+
+
+def _backend(gm: torch.fx.GraphModule, example_inputs):
+    """Generic backend which creates and preloads a launchable."""
+    # Export the graph module to mlir.
+    gm = turbine_cpu_pass_pipeline(gm, example_inputs)
+    logger.debug("Traced Graph Module:\n%s", str(gm))
+    fx_importer = FxImporter()
+    # TODO: figure out how to create a backend based on a dynamo `ExportedProgram`.
```

Comment:
Is an `ExportedProgram` a separate thing? My mental model `torch.compile` -> `torch.fx.GraphModule` that might have graph breaks, `torch.export` -> `torch.fx.GraphModule` + exception is thrown if there are graph breaks.